.github/ISSUE_TEMPLATE/help_wanted.yml

```diff
@@ -0,0 +1,59 @@
+name: Help wanted
```

Comment:
On GLPI core, support is redirected on forums

Reply:
agree too